background.js

```diff
@@ -1,49 +1,82 @@
-browser.pageAction.onClicked.addListener(async () => {
-  await browser.tabs.executeScript({
-    code: `document.getElementsByClassName("download")[0].click();`,
-  });
+const showMessage = async message => {
+  await browser.tabs.executeScript({code: `
+    closeBtn = document.createElement("span");
```

Comment:
Minor:

Instead of creating a global variable, consider wrapping this code in [an IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE) to use a regular variable declaration :smile:  The same goes for `alertBox`